specification/ai/Face/models.common.tsp

```diff
@@ -0,0 +1,196 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@azure-tools/typespec-azure-core";
+
+using Azure.Core;
+using Azure.Core.Traits;
+using TypeSpec.Http;
+using TypeSpec.Rest;
+
+namespace Face;
+
+@doc("The request for list resources.")
+model ListRequestOptions {
+  @doc("List resources greater than the \"start\". It contains no more than 64 characters. Default is empty.")
+  @query
+  start?: string;
+
+  @doc("The number of items to list, ranging in [1, 1000]. Default is 1000.")
+  @query
+  top?: int32;
+}
```

Comment:
These should be query parameters named ~~start~~ `skip` and `top`. That's the recommendation, anyway.

Reply:
This would be breaking change to existing service. We should consider in future release.